addons/crm/views/assets.xml

```diff
@@ -5,4 +5,9 @@
             <script type="text/javascript" src="/crm/static/src/js/tours/crm.js"></script>
         </xpath>
     </template>
+    <template id="assets_backend" name="CRM assets" inherit_id="web.assets_backend">
```

Comment:
I'm never really sure about the naming of those but I feel we can't go wrong with more info
```suggestion
    <template id="assets_backend" name="CRM assets backend" inherit_id="web.assets_backend">
```